internal/testutils/testhelpers/bubblewrap-runner/bubblewrap-runner.go

```diff
@@ -0,0 +1,93 @@
+// This is a small wrapper around the gpasswd command, which allows to run
+// gpasswd on a group file that's not at /etc/group. It does this by:
+//   - Reading the path to the group file from the GROUP_FILE environment variable.
+//   - Running the gpasswd command in a bubblewrap sandbox with the group file
+//     bind-mounted to /etc/group.
```

Comment:
Looks like this file is not gpasswd-specific anymore, but the comment still mentions gpasswd

Reply:
Yeah, actually we can even drop the whole file as separate executable to keep things easier.